nimare/diagnostics.py

```diff
@@ -288,15 +312,27 @@ def _transform(self, expid, label_map, result):
             target_value_map = "est"
         elif "stat" in result.maps:
             target_value_map = "stat"
+        elif "stat_desc-group1MinusGroup2" in result.maps:
+            target_value_map = "stat_desc-group1MinusGroup2"
+        elif "z_desc-specificity" in result.maps:
+            target_value_map = "z_desc-specificity"
         else:
             target_value_map = "z"
```

Comment:
this could be replaced by taking the union of two sets? one set being ("est", "stat", "stat_desc-group1MinusGroup2", "z_desc-specificity") and the other being the result.maps, then if the union of the sets is empty, have the target_value_map be "z".

Or taking a step back, the purpose of this is to find the uncorrected values, correct? (if the target_image was the corrected values, so one could theoretically manipulate the target_image name to create the target_value_map name, then it would just be a parameter instead of a list of if statements.